persistent/test/Database/Persist/TH/SharedPrimaryKeyImportedSpec.hs

```diff
@@ -28,7 +28,7 @@ import Database.Persist.TH.SharedPrimaryKeySpec (User, UserId)
 
 mkPersistWith sqlSettings $(discoverEntities) [persistLowerCase|
 
-Profile
+ProfileX
```

Comment:
`persistent` now fails to generate code for `Profile` because `Profile` is already a table defined in `discoverEntities`.